lang/python/python-yaml-3/Makefile

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-yaml-3
```

Comment:
Why?

PyYAML is already packaged in OpenWrt.
https://github.com/openwrt/packages/blob/master/lang/python/python-yaml/Makefile

Reply:
It is an older version. It is either this or patching docker-compose to work with newer PyYAML